src/dcap_provider.cpp

```diff
@@ -777,56 +875,66 @@ static std::string get_issuer_chain_cache_name(std::string url)
 static quote3_error_t get_collateral(
     CollateralTypes collateral_type,
     std::string url,
-    const char *issuer_chain_header,
+    const char* issuer_chain_header,
     std::vector<uint8_t>& response_body,
     std::string& issuer_chain,
     const std::string* const request_body = nullptr)
 {
+    std::unique_ptr<curl_easy> curl_operation;
     quote3_error_t retval = SGX_QL_ERROR_UNEXPECTED;
     std::string friendly_name = get_collateral_friendly_name(collateral_type);
     try
     {
         std::string issuer_chain_cache_name = get_issuer_chain_cache_name(url);
         if (auto cache_hit_collateral = try_cache_get(url))
         {
-            if (auto cache_hit_issuer_chain = try_cache_get(issuer_chain_cache_name))
+            if (auto cache_hit_issuer_chain =
+                    try_cache_get(issuer_chain_cache_name))
             {
                 log(SGX_QL_LOG_INFO,
                     "Fetching %s from cache: '%s'.",
                     friendly_name.c_str(),
                     url.c_str());
                 response_body = *cache_hit_collateral;
-                issuer_chain = std::string(cache_hit_issuer_chain->begin(), cache_hit_issuer_chain->end());
+                issuer_chain = std::string(
+                    cache_hit_issuer_chain->begin(),
+                    cache_hit_issuer_chain->end());
                 return SGX_QL_SUCCESS;
             }
         }
-
         log(SGX_QL_LOG_INFO,
             "Fetching %s from remote server: '%s'.",
             friendly_name.c_str(),
             url.c_str());
 
-        const auto curl_operation = curl_easy::create(url, request_body);
+        curl_operation = curl_easy::create(url, request_body);
         curl_operation->perform();
         response_body = curl_operation->get_body();
-        auto get_issuer_chain_operation =
-            get_unescape_header(*curl_operation, issuer_chain_header, &issuer_chain);
-
+        auto get_issuer_chain_operation = get_unescape_header(

```

Comment:
Nit: Personal preference but I think this (probably autoformatting?) made it worse 😆 -- The previous way was more readable for me. Here and below.

Reply:
i will try to revert it back and remove all formating changes.